cms/tests/page.py

```diff
@@ -112,6 +114,25 @@ def test_create_page_api(self):
         self.assertEqual(list(Title.objects.drafts().values_list('path', flat=True)), [u''])
         self.assertEqual(list(Title.objects.public().values_list('path', flat=True)), [u''])
 
+    def test_delete_page_no_template(self):
+        page_data = {
+            'title': 'root',
+            'slug': 'root',
+            'language': settings.LANGUAGES[0][0],
+            'template': 'nav_playground.html',
+
+        }
+        page = create_page(**page_data)
+        page.template = 'no_such_template.html'
+        signals.pre_save.disconnect(pre_save_page, sender=Page, dispatch_uid='cms_pre_save_page')
+        signals.post_save.disconnect(post_save_page, sender=Page, dispatch_uid='cms_post_save_page')
+        page.save(no_signals=True)
```

Comment:
The signal muting is only in the setup phase of the test (otherwise the invalid template is not accepted by the save())
The test core is https://github.com/divio/django-cms/pull/3935/files#diff-f7b86a7dc5f92dc7917bafd6e9069039R132 with signals restored